DEVELOPING.md

```diff
@@ -18,20 +18,34 @@ previous iteration of Dai was called Single Collateral Dai (SCD), or
 
 Dai has three different numerical units: `wad`, `ray` and `rad`
 
-- `wad`: fixed point decimal with 18 decimals (for basic quantities)
+- `wad`: fixed point decimal with 18 decimals (for basic quantities, e.g. balances)
 - `ray`: fixed point decimal with 27 decimals (for precise quantites, e.g. ratios)
-- `rad`: any number multiplied by a `ray`, using integer muliplication
+- `rad`: fixed point decimal with 45 decimals (result of integer multiplication with a `wad` and a `ray`)
 
 `wad` and `ray` units will be familiar from SCD. `rad` is a new unit and
-exists to prevent precision loss in the core CDP engine. Instead of
-using fixed point multiplication (e.g. `rmul`), integer multiplication is used.
+exists to prevent precision loss in the core CDP engine.
+
+The base of `ray` is `ONE = 10 ** 27`.
+
+A good explanation of fixed point arithmetic can be found at [Wikipedia](https://en.wikipedia.org/wiki/Fixed-point_arithmetic).
+
+## Multiplication
 
 Generally, `wad` should be used additively and `ray` should be used
-multiplicatively. It usually doesn't make sense to multiply a wad by a
-wad.
+multiplicatively. It usually doesn't make sense to multiply a `wad` by a
+`wad` (or a `rad` by a `rad`).
```

Comment:
No, you can multiply a rad by a rad just fine, e.g. a price multiplied by a ratio.